main/test/flix/experimental/Test.Dec.AssocType.flix

```diff
@@ -43,7 +43,7 @@ mod Test.Dec.AssocType {
     }
 
     instance Iterable2[String] {
-        type Eff[String] = Pure
```

Comment:
@mlutze Can we drop the experimental folder? Or at least the stuff with assoc  types?